src/redux/reducers/store.js

```diff
@@ -4,21 +4,31 @@ import reduxThunk from 'redux-thunk';
 import logger from 'redux-logger';
 import rootReducer from './index';
 
+export const STORE_VERSION = '1';
+export const CURRENT_STORE_VERSION = 'currentStoreVersion';
```

Comment:
Isn't the current store version equal to 1 instead of 'currentStoreVersion'?

Reply:
`STORE_VERSION` is the constant keeping track of the version in the code. `CURRENT_STORE_VERSION` is the constant for retrieving the `current version` in `local storage` and does not change.
Does this make sense?

Reply:
What is the name of the parameter that localStorage.getItem takes?

Reply:
`currentStoreVersion`

Reply:
Is that what the API docs say about the parameter that getItem takes?

Reply:
From the docs,  
`The getItem() method of the Storage interface, when passed a key name, will return that key's value, or null if the key does not exist, in the given Storage object.`

Kindly clarify on what I am missing.

Reply:
Your constant name should reflect that too.

Reply:
Noted. Updated `CURRENT_STORE_VERSION` to `CURRENT_STORE_VERSION_KEY`.
Thanks for pointing that out.

Reply:
You have hit the nail on the head. That is exactly what i was looking for. 😊